docs/profiling.md

```diff
@@ -0,0 +1,31 @@
+# vHive profiling tool
```

Comment:
please split the long lines (80-100 characters are MAX for a single line)

Reply:
Done